modules/gitops-release-notes-1-6-2.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-6-2_{context}"]
+= Release notes for {gitops-title} 1.6.2
+
+{gitops-title} 1.6.2 is now available on {product-title} 4.8, 4.9, 4.10 and 4.11.
+
+[id="fixed-issues-1-6-2_{context}"]
+== Fixed issues
+
+The following issue have been resolved in the current release:
+
+* Before this update, subscription health check was marked *degraded* for missing *InstallPlan* when more than 5 operators were installed in a project. This update fixes the issue. link:https://issues.redhat.com/browse/GITOPS-2018[GITOPS-2018]
+
+* Before this update, the {gitops-title} Operator would spam the cluster with deprecation notice warning whenever it detected that an Argo CD instance used deprecated fields. This update fixes this issue and shows only one warning event for each instance that detects a field. link:https://issues.redhat.com/browse/GITOPS-2230[GITOPS-2230]
```

Comment:
```suggestion
* Before this update, the {gitops-title} Operator would spam the cluster with a deprecation notice warning whenever it detected that an Argo CD instance used deprecated fields. This update fixes this issue and shows only one warning event for each instance that detects a field. link:https://issues.redhat.com/browse/GITOPS-2230[GITOPS-2230]
```

Reply:
In the current description, the _before_ and _after_ sound very similar: one warning for one instance. I wonder if the difference can be made clearer?

Reply:
For clarity and localization, please use articles in sentences. Avoid "telegraphic" style.